docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -0,0 +1,117 @@
+# List of Usability Tasks
+
+## Users
+
+As a user, you are a member of the general public who is using the Kukarella web application; you are someone who would
+like to have a more immersive text-to-speech experience where an animated avatar reads user text aloud.
+
+In your pursuit of having an animated text-to-speech experience, you will use our team's premium Kukarella account, which was provided by the client.
+
+### User Scenario 1: Language Test
+
+1. On the Kukarella web app
+
+    1. Enter text in a language other than English (you may use Google Translate, but your feedback will be more
+    valuable if you are familiar with the language)
+    2. Choose a voice for that language
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different language each time.
+
+### User Scenario 2: Voice Test
+
+1. On the Kukarella web app
+
+    1. Enter text in a language
+    2. Select a voice for that language (try a different one each time you repeat this scenario)
+    3. Convert your input to a .wav and .txt file
+    4. Inform the examiner which avatar you'd like to use: Barb or Boy
+
+2. Let the examiner do the back-end work that would happen if the API was integrated
+    a. Send the .wav file, .txt file, language, and avatar selection to the API
+    b. Wait for the API to generate the video
+
+3. Watch the video
+
+Repeat this scenario with a different voice each time.
+
+### User Scenario 3: Voice Effects
+
+1. On the Kukarella web app
+
+    1. Enter text in a language
+    2. Select a voice for that language that supports effects
+    3. Play around with the following effects to see if the animated avatar looks natural with each:
+        - Pitch
+        - Pause
+        - Speed
+        - Say As
+        - Emphasis
+        - Volume
+        - Mute
+        - Breath
+        - Whisper
```

Comment:
Is this an option on the web app?

Reply:
IT IS!!! I just tried it this morning!

Reply:
😮 woah that's so fun! We should def be testing to see how rhubarb handles those!!